scirpy/io/_io_bdrhapsody.py

```diff
@@ -0,0 +1,96 @@
+import pandas as pd
+import numpy as np
+import scirpy as ir 
+import anndata
+import scanpy as sc
+
+class ImportBDrhapsodyData:
+
+    def load_tcr_seq(data):
+
+        tcr_table = pd.read_csv(data, sep=",", index_col=None, na_values=["None"], true_values=["True"],comment='#')
+        tcr_table['productive_Alpha_Gamma'] = tcr_table.TCR_Alpha_Gamma_CDR3_Translation_Dominant.str.contains('\*', regex=True)
+        tcr_table['productive_Beta_Delta'] = tcr_table.TCR_Beta_Delta_CDR3_Translation_Dominant.str.contains('\*', regex=True)
+        print(tcr_table['productive_Beta_Delta'])
+        tcr_table
+        tcr_cells = []
+        for idx, row in tcr_table.iterrows():
+            cell = ir.io.AirrCell(cell_id=row["Cell_Index"])
+            alpha_chain = ir.io.AirrCell.empty_chain_dict()
+            beta_chain = ir.io.AirrCell.empty_chain_dict()
+            alpha_chain.update(
+                {
+                    "locus": "TRA",
```

Comment:
I think we discussed this already, but there's no way to find out if this is a alpha or a gamma chain?

Reply:
As far as i know from may data not